packages/tooltip/src/use-tooltip.ts

```diff
@@ -117,29 +117,30 @@ export function useTooltip(props: UseTooltipProps = {}) {
   const exitTimeout = React.useRef<number>()
 
   const openWithDelay = React.useCallback(() => {
-    if (!isDisabled) {
+    if (!isDisabled && !enterTimeout.current) {
```

Comment:
Do not set the enter timer if it still exists to avoid rendering multiple tooltips